### DISCUSSION TITLE
54112 4 wire just won't work

### DISCUSSION THREAD

**Discussion by nono721987** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#589] [Q&A]
Hello I am from Germany. excuse my not so good english. I've tried in vain to recreate the project, but now I can't get any further. i have nodemcu v3 ,pcb v2b, TXS0108E and in my opinion soldered everything correctly. all connections have been measured several times with a multimeter and they are all present. I've tried a second board. even without a level shifter. I have the current version 4.1.4. in the settings I took the board 54123 and the board pcbv2b. then swapped the custom and the pins with each other. in a certain constellation, the led on the board also flashed, which means that there is communication. however, the dsp and the cio do not transmit any data to the web interface. I then entered the pins from the dsp in the cio and then something happened in the web interface, but it was all weird. some buttons were on and a temperature was displayed, but I couldn't change any button. but the dsp showed nothing except 00. it would be really great if someone who has the same pump as me could show me their settings and the wiring. it would be super annoying to put it all away again.

**Comment by tunnus-1** [comment]
See this [post](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/issues/510), you could try an older fw version to see if it makes any difference

**Comment by Ginoooke** [comment]
Did you manage to get this working?

**Reply by tunnus-1** [reply]
Yes, actually I did by using v4.2.5 and NO54154 configuration and PCB v2. Having Paris 4-wire model. Btw, @visualapproach you could update this information to user manual.

**Reply by tunnus-1** [reply]
... @visualapproach, bought you a coffee, thanks for this project!

**Reply by visualapproach** [reply]
> ... @visualapproach, bought you a coffee, thanks for this project! 

Yes I got a notification on that, thank you! A fellow Swede judging from the name? People like you make it worth the effort!

**Reply by visualapproach** [reply]
> Yes, actually I did by using v4.2.5 and NO54154 configuration and PCB v2. Having Paris 4-wire model. Btw, @visualapproach you could update this information to user manual. 

Sure, but I may be slow here. What do you want me to update in the manual? That it works with 54154 or with V2? Or something else?

**Reply by tunnus-1** [reply]
I mean that "table of models":

![Screenshot 2024-05-11 at 23 58 33](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/74300867/35ee2b04-7059-49dc-81ca-d323f113543b)
